### PR TITLE
Notify case of PRINT_CASE_SELECTED on fulfilment

### PR DESCRIPTION
**Pull request by NickGrantONS**
# Motivation and Context
We want to know what pack code has been used on a fulfilment request.

# What has changed
Send a message to Case Processor when we print an adhoc fulfilment request.

# How to test?
Run the ATs against the same branch name

# Links
Trello: https://trello.com/c/3xvFQbvw

### CODE REVIEW

**Comment by NickGrantONS** [comment]
[Case event, not being recorded stating that case selected for print pack..... for a fulfillment request.](https://trello.com/c/3xvFQbvw/1062-case-event-not-being-recorded-stating-that-case-selected-for-print-pack-for-a-fulfillment-request)

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -42,7 +42,6 @@ queueconfig:
   outbound-field-queue: Action.Field
   outbound-exchange: action-outbound-exchange
   action-case-exchange: action-case-exchange
-  action-case-queue: action.events
```
Comment (ryangrundy7) [inline src/main/resources/application.yml]: Why has this queue been removed?

Comment (NickGrantONS) [reply src/main/resources/application.yml]: in the production code, we publish to an exchange, not a queue

**Comment by NickGrantONS** [comment]
Have made logged descriptions more generic in Case Processor

File: src/main/resources/application.yml (hunk 1 above)
Comment (ryangrundy7) [reply src/main/resources/application.yml]: Does it still need to be in the definitions.json file then?

**Pull request merged**